static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js

```diff
@@ -131,6 +138,11 @@ define([
                 players[atomId].endSlate = getEndSlate(overlay);
             }
         }
+
+        if (closest(players[atomId].iframe, '.immersive-main-media__media')) {
```

Comment:
might be nice to pull this out as a var for clarity e.g. 
`var immersiveYouTubeIframe = players[atomId].iframe, '.immersive-main-media__media';`